Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "viam-rust-utils"
-version = "0.3.0"
+version = "0.2.17"
```

Comment:
this got accidentally bumped to `0.3.0` in testing. I'm just setting it back to where it's supposed to be.